bin/local-pelican-site.sh

```diff
@@ -73,7 +73,8 @@ then
   cd $IP && git pull > /dev/null && cd .. 
 else
   echo "Cloning $IP"
-  git clone $GH/$IP 2>&1 
+  # Sometimes useful to add -b <branch> for buildsite testing
+  git clone -b sitemap-plugin $GH/$IP 2>&1 
```

Comment:
This may be true for your development, but is certainly not true in production,

Reply:
resolved here:https://github.com/apache/infrastructure-pelican/pull/39/commits/a96c78cbb6a590816da1df0f059a8fbd0c2d15d3